docs/machine-learning/resources/ml-net-cli-telemetry.md

```diff
@@ -37,16 +37,16 @@ Opt out of the telemetry feature by setting the `MLDOTNET_CLI_TELEMETRY_OPTOUT`
 
 The feature collects the following data:
 
-- What command was invoked, such as `auto-train`
-- Command-line parameter names used (that is, "dataset-name, label-column-name, ml-task, output-path, max-exploration-time, verbosity")
+- What command was invoked, such as `classification`
+- Command-line parameter names used (that is, "dataset, label-col, output-path, train-time, verbosity")
 - Hashed MAC address: a cryptographically (SHA256) anonymous and unique ID for a machine
 - Timestamp of an invocation
 - Three octet IP address (not full IP address) used only to determine geographical location
 - Name of all arguments/parameters used. Not the customer's values, such as strings
 - Hashed dataset filename
 - Dataset file-size bucket
 - Operating system and version
-- Value of --task parameter: Categorical values, such as `regression`, `binary-classification`, and `multiclass-classification`
+- Value of ML task commands: Categorical values, such as `regression`, `classification`, and `recommendation`
```

Comment:
Is this the same as the first bullet point? If so, I would replace the first one with this.